app/utils/interfaces.ts

```diff
@@ -2,13 +2,15 @@ import { Event, stringOrDate } from 'react-big-calendar';
 
 export interface Attendees {
   attendee: {
+    name: string;
     email: string;
   };
 }
 export interface CalEvent extends Event {
   id?: number;
   description?: string;
   location?: string;
+  type?: 'public' | 'private';
   ownerId?: number;
   calendarId?: number;
   attendees?: Attendees[];
```

Comment:
its `visibility` not `type`. It is already present in the latest `develop` branch. Please take a pull once

Reply:
> its `visibility` not `type`. It is already present in the latest `develop` branch. Please take a pull once

Adding to this why have we kept `type` to be optional? 
Should it not be required always as an Event cannot be made without it? And we need it to display the event type inside the `event details` component.

Reply:
Yes, fixed it. 

Note: `visibility` was also optional when I pull latest `develop ` branch.

Reply:
> Yes, fixed it.
> 
> Note: `visibility` was also optional when I pull latest `develop ` branch.

Thanks for the note. Everything is kept optional for an Event as it can be created with just the `title`, `from` and `to`. However, by default, an event will be private unless the user makes it public. So, we can go ahead and change `visibility` from optional to required. Can you please do that?